environment.yml

```diff
@@ -1,16 +1,5 @@
-name: django
+name: wam-django
```

Comment:
I found `django` to be too broad...

Reply:
I agree, but will this have an effect on alreqady installed instances of WAM (i.e. our server?)

Reply:
The environment name `django` is also used in the `docker-compose.yml` file. Although we could also change `django` to `wam-django`, I decided to revert this change for the time being (a41ce1ced6292227342befe52ef9720f2ff52670)